src/LoginProviders/LoginProvider.php

```diff
@@ -82,15 +82,15 @@ protected function detectRememberMe()
         if (!$this->isLoggedIn()) {
             $request = Request::current();
 
-            if ($request->cookie('lun') != "") {
+            if (isset($request->cookieData['lun'])) {
                 $username = $request->cookie('lun');
                 try {
                     $user = User::fromIdentifierColumnValue($username);
 
-                    $token = $request->cookie('ltk');
+                    $token = $request->cookieData['ltk'];
 
                     if ($user->validateToken($token)) {
-                        $this->LoggedIn = true;
```

Comment:
instead of this whole block of code, it should really just do `$this->forceLogin($user)`. I dunno why it didn't always do that. That's what that method is for! That way, if someone is using their own login provider with an overridden `forceLogin` method, that functionality will be applied when they use remember login, without having to override this method